File: app/src/main/java/net/tlalka/android/fiszki/models/dto/LanguagesDto.java
package net.tlalka.android.fiszki.models.dto;

import net.tlalka.android.fiszki.models.types.LanguageType;
import org.parceler.Parcel;

import java.util.List;

@Parcel
public class LanguagesDto {

    List<LanguageType> languages;

    public LanguagesDto() {
    }

    public LanguagesDto(List<LanguageType> languages) {
        this.languages = languages;
    }

    public List<LanguageType> getLanguages() {
        return languages;
    }
}

File: app/src/main/kotlin/net/tlalka/fiszki/model/dto/parcel/LanguagesDto.kt
package net.tlalka.fiszki.model.dto.parcel

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import net.tlalka.fiszki.model.types.LanguageType

@Parcelize
data class LanguagesDto constructor(var languages: List<LanguageType>) : Parcelable
